### PR TITLE
Implements GQL endpoints for CSA Keys

### PR DESCRIPTION
**Pull request by vyzaldysanchez**
Closes: https://app.shortcut.com/chainlinklabs/story/20532/create-gql-endpoints-for-csa-keys

This enables a GQL query to fetch the list of CSA Keys, and a GQL mutation to create a single CSA Key.

### CODE REVIEW

**Comment by cl-homu** [comment]
:umbrella: The latest upstream changes (presumably 7b7e6dbdb000ee6f6e041bced189ec9c88038874) made this pull request unmergeable. Please [resolve the merge conflicts](https://rustc-dev-guide.rust-lang.org/git.html#rebasing-and-conflicts).

File: core/web/resolver/csa_keys_test.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
+package resolver
+
+import (
+	"encoding/json"
```
Comment (jkongie) [inline core/web/resolver/csa_keys_test.go:4]: Organise imports in the order of stdlib/external/internal packages separated by a new line

File: core/web/schema/schema.graphql [hunk 2]
```diff
@@ -18,6 +19,8 @@ type Query {
 
 type Mutation {
     createBridge(input: CreateBridgeInput!): CreateBridgePayload!
+    # Creating a CSA Key requires no input, see csa_keys_controller.go:36
```
Comment (jkongie) [inline core/web/schema/schema.graphql:22]: You can remove this comment

**Review by jkongie — CHANGES_REQUESTED** [review]
Looks good! Just minor changes

**Review by makramkd — DISMISSED** [review]
Looks good, assuming James' comments are addressed!

File: core/web/resolver/csa_keys_test.go (hunk 1 above)
Comment (vyzaldysanchez) [reply core/web/resolver/csa_keys_test.go:4]: Noted, I've modified Goland to group all `github.com/smartcontractkit/` imports together, same as using `goimports -local github.com/smartcontractkit/`.

**Review by jkongie — DISMISSED** [review]

**Comment by cl-homu** [comment]
:umbrella: The latest upstream changes (presumably d96013ef2008bf6d23c4d9c26a7c2e60bc745717) made this pull request unmergeable. Please [resolve the merge conflicts](https://rustc-dev-guide.rust-lang.org/git.html#rebasing-and-conflicts).

**Review by jkongie — APPROVED** [review]

**Pull request merged**